sources/install.sh

```diff
@@ -1238,15 +1349,15 @@ function install_bloodhound_old_v3() {
   unzip /tmp/BloodHound-linux-x64.zip -d /opt/tools/
   mv /opt/tools/BloodHound-linux-x64 /opt/tools/BloodHound3
   rm /tmp/BloodHound-linux-x64.zip
-}
+}#
```

Comment:
why adding a `#` here?

Reply:
}# : this indicates this tool is duplicated and his version should not be documented because it would cause a duplication

Reply:
alright, we should probably add a comment indicating that then